PyART/analysis/match.py

```diff
@@ -40,9 +40,6 @@ def __init__(self,
             raise ValueError(f"Kind '{settings['kind']}' not recognized")
         
         if self.settings['cut']:
-            #tlen1 = WaveForm1.u[-1]-WaveForm1.u[0]
-            #tlen2 = WaveForm2.u[-1]-WaveForm2.u[0]
-            #DeltaT = tlen2-tlen1
             tmrg1,_,_,_ = WaveForm1.find_max()-WaveForm1.u[0]
             tmrg2,_,_,_ = WaveForm2.find_max()-WaveForm2.u[0]
             DeltaT = tmrg2-tmrg1
```

Comment:
It would be good to resize waveforms also if wf1 is longer than wf2, and not just if wf2 is longer. As things are, matcher(wfA, wfB) and matcher(wfB,wfA) are not giving the same result if cut=True, which is bad